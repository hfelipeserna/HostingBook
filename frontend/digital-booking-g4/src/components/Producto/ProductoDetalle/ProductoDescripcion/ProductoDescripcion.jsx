import TituloBloque from "../../../TituloBloque/TituloBloque";
import styles from "./ProductoDescripcion.module.css";

export default function ProductoDescripcion({
  alojamiento: { descripcion, ciudad },
}) {
  return (
    <section className={styles.bloqueProductoDescripcion}>
      <TituloBloque>{`Alojate en el coraz√≥n de ${ciudad.nombre}`}</TituloBloque>
      <p>{descripcion}</p>
      <div className={styles.bloqueProductoInfo}></div>
    </section>
  );
}
