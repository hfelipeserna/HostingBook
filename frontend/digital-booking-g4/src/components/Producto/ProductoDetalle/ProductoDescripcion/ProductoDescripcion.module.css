.bloqueProductoDescripcion {
    width: 100%;
}

.bloqueProductoDescripcion p {
    width: 100%;
    margin-top: 20px;
    font-size: 0,875rem;
    text-align: justify;
}

.bloqueProductoInfo {
    width: 100%;
    margin-top: 32px;
}