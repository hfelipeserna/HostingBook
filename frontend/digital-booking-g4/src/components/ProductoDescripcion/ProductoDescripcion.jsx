import TituloBloque from "../TituloBloque/TituloBloque";
import styles from "./ProductoDescripcion.module.css";
import useScreenWidth from "../../hooks/useScreenWidth";

export default function ProductoDescripcion({ alojamiento }) {

    return (
        <section className={styles.bloqueProductoDescripcion}>
            <>
                <TituloBloque> {alojamiento.product.title} </TituloBloque>
                <div className={styles.bloqueProductoInfo}>
                    {alojamiento.product.info.map((parrafo,i) => {
                        return <p key={i}>{parrafo}</p>
                    })}
                </div>
            </>
        </section>
    );
}