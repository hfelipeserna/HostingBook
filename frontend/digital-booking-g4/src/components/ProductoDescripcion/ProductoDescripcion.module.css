.bloqueProductoDescripcion {
    width: 100%;
    margin-bottom: 58px;
}

.bloqueProductoDescripcion p {
    width: 100%;
    margin-bottom: 20px;
    font-size: 0,875rem;
}

.bloqueProductoInfo {
    width: 100%;
    margin-top: 32px;
}