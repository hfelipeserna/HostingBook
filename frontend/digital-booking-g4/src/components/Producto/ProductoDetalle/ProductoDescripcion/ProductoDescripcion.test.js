import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import ProductoDescripcion from "./ProductoDescripcion"
describe("Producto drescripcion  test", function () {
    const alojamiento =
    {
        descripcion: "esta es la descriopcion",
        ciudad: {
            nombre: "este es el nombre de la ciudad"
        }
    }
    test("Renderizado Por defaulto", () => {
        const renderizacionSimple = render(<ProductoDescripcion alojamiento={alojamiento}></ProductoDescripcion>)
        expect(renderizacionSimple).toMatchSnapshot()
    })
})