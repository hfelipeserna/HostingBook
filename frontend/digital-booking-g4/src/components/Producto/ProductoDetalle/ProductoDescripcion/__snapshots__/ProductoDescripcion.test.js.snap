// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Producto drescripcion  test Renderizado Por defaulto 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="bloqueProductoDescripcion"
      >
        <h3
          class="subtituloBloque undefined"
        >
          Alojate en el corazón de este es el nombre de la ciudad
        </h3>
        <p>
          esta es la descriopcion
        </p>
        <div
          class="bloqueProductoInfo"
        />
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="bloqueProductoDescripcion"
    >
      <h3
        class="subtituloBloque undefined"
      >
        Alojate en el corazón de este es el nombre de la ciudad
      </h3>
      <p>
        esta es la descriopcion
      </p>
      <div
        class="bloqueProductoInfo"
      />
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
